pkg/kubectl/cmd/edit.go

```diff
@@ -120,12 +120,18 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		return err
 	}
 
+	// NB: we use `f.Decoder(false)` to get a plain deserializer for
```

Comment:
I would, but I didn't want to have a large block comment in the middle of a struct instantiation.  WDYT?